Gemfile

```diff
@@ -29,7 +28,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'rubocop', require: false
```

Comment:
`Should i correct all of the Rubocop ofenses?.`

No, we'll do it on the way.

`There are several that we can modify. Like "Line too long" (should it be allways 80?) or "missing top-level class documentation comment", in that case, do i modify them? or should we stick to rubocop's suggestions strictly?`

Not all of us should strictly follow them, we could come to an agreement later.